index.html

```diff
@@ -57,6 +59,169 @@ <h1 id="home-section-title">I am Hiro Glad to See You!</h1>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="author-skills">Back End Dev</p>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="card-date">2015</p>
+            </div>
+            <p class="card-inner-info">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <div class="skills">
+              <button class="card-skills-button">Html</button>
+              <button class="card-skills-button">Css</button>
+              <button class="card-skills-button">Javascript</button>
+            </div>
+            <button class="see-project-button">See Project</button>
+          </div>
+
+          <div class="work-section-card">
+            <div class="img-container">
+              <img src="/img/Snapshoot3Portfolio.png" alt="Work" />
+            </div>
+            <h1 class="author-name">Tonic</h1>
+            <div class="card-info">
+              <p class="card-info-title">Canopy</p>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="author-skills">Back End Dev</p>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="card-date">2015</p>
+            </div>
+            <p class="card-inner-info">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <div class="skills">
+              <button class="card-skills-button">Html</button>
+              <button class="card-skills-button">Css</button>
+              <button class="card-skills-button">Javascript</button>
+            </div>
+            <button class="see-project-button">See Project</button>
+          </div>
+
+          <div class="work-section-card">
+            <div class="img-container">
+              <img src="/img/Snapshoot4Portfolio.png" alt="Work" />
+            </div>
+            <h1 class="author-name">Multi-Post-Stories</h1>
+            <div class="card-info">
+              <p class="card-info-title">Canopy</p>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="author-skills">Back End Dev</p>
+              <img src="/img/Counter.png" alt="Counter" />
+              <p class="card-date">2015</p>
+            </div>
+            <p class="card-inner-info">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+
+            <div class="skills">
+              <button class="card-skills-button" type="submit">Html</button>
+              <button class="card-skills-button" type="submit">Css</button>
+              <button class="card-skills-button" type="submit">Javascript</button>
+            </div>
+            <button class="see-project-button" type="submit">See Project</button>
+          </div>
+        </div>
```

Comment:
- [ ] __For listing of projects, please use the `ul` tag and use `li` for each project__
Using the `ul` and `li` tags will make it obvious for screen readers that they belong to the same group. It also helps with proper accessibility and optimization of the webpage

Reply:
when I am trying to use the ul and li i am getting the linter error that is saying your ul should not have divs elements so I do know how to fix that

Reply:
Have you tried a `p` tag?

Reply:
I have left a comment on how to fix this issue.